documentation-website/src/components/header.tsx

```diff
@@ -81,7 +81,7 @@ const Header = ({
         label={'slack'}
       />
       <SocialLink
-        to="https://github.com/Idrinth/api-bench"
+        to="https://github.com/idrinth/api-bench"
```

Comment:
This is one of the links that actually has to be updated

Reply:
Done